index.js

```diff
@@ -121,22 +124,21 @@ Liftoff.prototype.buildEnvironment = function (opts) {
     }, this);
   }
 
-  // use rechoir to autoload any required modules
-  var autoloads;
-  if (configPath) {
-    autoloads = rechoir.prepare(this.extensions, configPath, cwd, true);
-    if (autoloads instanceof Error) {
-      autoloads = autoloads.failures;
-    }
-    if (Array.isArray(autoloads)) {
-      autoloads.forEach(function (attempt) {
-        if (attempt.error) {
-          this.emit('requireFail', attempt.moduleName, attempt.error);
-        } else {
-          this.emit('require', attempt.moduleName, attempt.module);
+  registerLoader(this.extensions, configPath, cwd, this);
+
+  var configFiles = {};
+  if (isPlainObject(this.configFiles)) {
+    var exts = this.extensions, notfound = { path: null }, eventEmitter = this;
+    configFiles = mapValues(this.configFiles, function(prop, name) {
+      var defaultObj = { name: name, cwd: cwd, extensions: exts };
+      return mapValues(prop, function(pathObj) {
```

Comment:
Is 2-level nested objects something we actually want to support?  /cc @tkellen (example usage https://github.com/js-cli/js-liftoff/pull/67/files#diff-910eb6f57886ca16c136101fb1699231R326)